bolt-sidecar/src/api/commitments/spec.rs

```diff
@@ -93,6 +97,9 @@ impl From<CommitmentError> for JsonRpcError {
             CommitmentError::InvalidParams(err) => Self::new(-32602, err.to_string()),
             CommitmentError::Internal => Self::new(-32603, err.to_string()),
             CommitmentError::RejectedJson(err) => Self::new(-32604, err.to_string()),
+            CommitmentError::ValidatorNotAvailable(ref pk) => {
+                Self::new(600, err.to_string()).with_data(json!(pk))
```

Comment:
where does 600 come from? is it our custom spec?

Reply:
Yes. I know that it's not yet public but I thought it would be a good idea to start using it